src/Hipace.cpp

```diff
@@ -245,7 +246,7 @@ void
 Hipace::MakeNewLevelFromScratch (
     int lev, amrex::Real /*time*/, const amrex::BoxArray& ba, const amrex::DistributionMapping&)
 {
-    SetMaxGridSize(1048576); // 2^20, this ensures that level 1 uses only 1 box longitudinally.
+    // SetMaxGridSize(1048576); // 2^20, this ensures that level 1 uses only 1 box longitudinally.
```

Comment:
Why did you comment this out?

Reply:
I should have removed it altogether, this is a leftover. Let me clean that